site/assets/js/map.js

```diff
@@ -1,3 +1,5 @@
+var g, svg;
```

Comment:
can we put these on a maps object instead of tossing them into the global namespace? I can play around with that refactor a bit. non-blocker.